airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/templates/test_report.html.j2

```diff
@@ -178,11 +178,11 @@ function copyToClipBoard(htmlElement) {
       <div class="content-inner">
         {% if step_result.stdout %}
         <span class="std">Standard output(<button onclick="copyToClipBoard(this)">Copy to clipboard</button>):</span>
-        <pre>{{ step_result.stdout }}</pre>
+        <pre>{{ step_result.html_escaped_stdout }}</pre>
```

Comment:
```suggestion
        <pre>{{ step_result.stdout | e }}</pre>
```
[I think jinja has a builtin escaping mechanism. ](https://jinja.palletsprojects.com/en/3.0.x/templates/#working-with-manual-escaping). I'd find it simpler to use it instead of adding new properties on the `Result` class.